data/init.sql

```diff
@@ -19,7 +19,7 @@ CREATE TABLE IF NOT EXISTS users (
 	student_id INT UNSIGNED,
 	created_at TIMESTAMP,
 	updated_at TIMESTAMP,
-	UNIQUE (email)
+	UNIQUE (email, student_id)
 );
```

Comment:
ok wait a sec does this mean that each pair of email and studentID has to be unique? i.e. its ok to have the same student ID as long as the email is different and vice versa?